jazzhands/__main__.py

```diff
@@ -141,23 +207,32 @@ def main(argv):
         )
         if os.path.isdir(dirname) and has_settings:
             project_dir = dirname
-    assert project_dir
+    assert project_dir  # If we can't find the location of the project's "main" package, abort
```

Comment:
This would be friendlier if it showed a message explaining the problem (how jazzhands locates the "main" package, and that it couldn't find it).